helm/values.yaml

```diff
@@ -118,7 +118,7 @@ signaleringen:
   restartPolicy: Never
   image:
     repository: curlimages/curl
-    tag: latest
+    tag: latest@sha256:c1fe1679c34d9784c1b0d1e5f62ac0a79fca01fb6377cdd33e90473c6f9f9a69
```

Comment:
We probably should not use latest here right?

Reply:
I guess so but these pins are supported in Helm Charts (and/or our provisioning scripts)?

Reply:
Agreed, this should add a proper version belonging to this digest

```suggestion
    tag: 8.11.1@sha256:c1fe1679c34d9784c1b0d1e5f62ac0a79fca01fb6377cdd33e90473c6f9f9a69
```